email/1.3/modules/ROOT/edits/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+
+. Select the *On New Email - IMAP* or *On New Email - POP3* source in the Studio canvas.
+. In the source configuration screen, enable the *Delete after retrieve* parameter.
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - IMAP* source:
+
+.Delete after retrieve with On New Email - IMAP operation
+image::email-delete-1.png[Delete after retrieve with On New Email - IMAP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-imap` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+		<email:listener-imap config-ref="Email_IMAP" watermarkEnabled="false" deleteAfterRetrieve="true">
+			<scheduling-strategy >
+				<fixed-frequency />
+			</scheduling-strategy>
+		</email:listener-imap>
+	</flow>
+----
+
+The following screenshot shows the *Delete after retrieve* configuration for the *On New Email - POP3* source:
+
+.Delete after retrieve with On New Email - POP3 operation
+image::email-delete-2.png[Delete after retrieve with On New Email - POP]
+
+In the XML editor, the configuration `deleteAfterRetrieve="true"` is under the `listener-pop3` tag:
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-POP3">
+  <email:listener-pop3 config-ref="Email_POP3" deleteAfterRetrieve="true">
+    <scheduling-strategy >
+      <fixed-frequency />
+    </scheduling-strategy>
+  </email:listener-pop3>
+</flow>
+----
+
+
+== Configure the Enable Watermark Parameter
+
+Email Connector can watermark emails over the IMAP protocol by filtering emails based on their received date. The *On New Email - IMAP* source supports the *Enable Watermark* parameter that enables to dispatch only messages containing emails received after the last poll executed.
+
+The following example shows how to configure this parameter in Anypoint Studio:
+
+. Select the *On New Email - IMAP* source in the Studio canvas.
+. On the source configuration screen, enable the *Enable Watermark* parameter.
+
+The following screenshot shows this parameter configuration for the source:
+
+.Enable Watermark with On New Email - IMAP source
+image::email-watermark-1.png[Enable Watermark with On New Email - IMAP]
+
+In the XML editor, the configuration `watermark="true"` is under the `listener-imap` tag:
+
+[source, xml, linenums]
+----
+<flow name="OnNewEmail-IMAP">
+    <email:listener-imap config-ref="Email_IMAP" watermark="true">
+        <scheduling-strategy>
+            <fixed-frequency/>
+        </scheduling-strategy>
+    </email:listener-imap>
+</flow>
+----
+
+== Configure IMAP Matcher and POP3 Matcher Parameters
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support *Imap matcher* and *Pop3 matcher* parameters that enable you to configure custom matchers  and provide another way of filtering dispatched messages. Each source matcher can filter emails by dates, addresses, subject, flags, and so on.
```

Comment:
```suggestion
Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support *Imap matcher* and *Pop3 matcher* parameters that enable you to configure custom matchers and thereby provide another way of filtering dispatched messages. Each source matcher can filter emails by dates, addresses, subject, flags, and so on.
```
"dispatched messages" or "received messages"?
What are the message states; what is dispatching them originally, receiving them, dispatching them again to be processed, deleted if not watermarked, processed?